src/components/login/loginForm.jsx

```diff
@@ -0,0 +1,224 @@
+import jwtDecode from 'jwt-decode';
+import Paper from '@mui/material/Paper';
+import React, { useState } from 'react';
+import axios from 'axios';
+import Box from '@mui/material/Box';
+import { useNavigate } from 'react-router-dom';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+
+import { withFormik } from 'formik';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  withStyles,
+  Card,
+  CardContent,
+  CardActions,
+  TextField,
+  Typography,
+  Button,
+} from '@material-ui/core';
+import { thisUser, logginUser } from '../../redux/features/auth/loginSlice';
+import {
+  selectLogginStatus,
+  loginMode,
+} from '../../redux/features/auth/isLoggedSlice';
+
+import google from '../../asset/icons8-google.svg';
+import facebook from '../../asset/icons8-facebook.svg';
+
+import '../../styles/registration.scss';
+import colors from '../../styles/colorVariables';
+
+const styles = () => ({
+  card: {
+    maxWidth: 420,
+    marginTop: 100,
+  },
+  container: {
+    display: 'Flex',
+    justifyContent: 'center',
+  },
+  actions: {
+    justifyContent: 'center',
+  },
+});
+
+const form = (props) => {
+  const dispatch = useDispatch();
+  const [email, setemail] = useState();
+
+  const navigate = useNavigate();
+
+  const authenticated = useSelector(thisUser);
+  const isLogged = useSelector(selectLogginStatus);
+
+  const getEmail = (e) => {
+    setemail(e.target.value);
+  };
+  const [password, setpassword] = useState();
+  const getPassword = (e) => {
+    setpassword(e.target.value);
+  };
+  const showPassword = () => {
+    const x = document.getElementById('password');
+    if (x.type === 'password') {
+      x.type = 'text';
+    } else {
+      x.type = 'password';
+    }
+  };
+
+  const googleFun = () => {
+    window.open('https://barefoot.onrender.com/api/auth/google', '_blank');
+  };
+
+  const facebookFun = () => {
+    window.open('https://barefoot.onrender.com/api/auth/facebook', '_blank');
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post('https://barefoot-nomad-v2.herokuapp.com/api/auth/login', {
+        email,
+        password,
+      })
```

Comment:
configure Axios globally. with some basic headers that are being used ex: token.